### PR TITLE
Documentation for asset diffing endpoint

### PR DESCRIPTION
**Pull request by nickcurie**
Initial documentation for the asset diff API

### CODE REVIEW

**Comment by teevans** [comment]
@jjarrett21 - Take a look at this documentation and make sure that we can facilitate the UI that @Annalillie has built for this. I've got you slated to implement this week after next.

File: asset-diff.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+```
+
+## Example API Response
+
+```
+{
+    ...
+    Entity: // this is just a typical asset
+        {
```
Comment (michaelmdresser) [inline asset-diff.md:47]: I think this is actually an array, not an object, right? So it looks more like this:

```
[
  ...,
  { "Entity": {}, "Kind": "added/removed" }
]
```

File: asset-diff.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+http://localhost:9090/model/assets/diff?before=yesterday&after=today
+
+http://localhost:9090/model/assets/diff?before=1d offset 5d&after=1d
+
+http://localhost:9090/model/assets/diff?before=lastmonth&after=month
+
+http://localhost:9090/model/assets/diff?before=2022-07-01T00:00:00Z,2022-07-02T00:00:00Z&after=2022-07-06T00:00:00Z,2022-07-07T00:00:00Z
+
+```
```
Comment (michaelmdresser) [inline asset-diff.md:39]: Adding a little comment explaining each example would be helpful! Something like:

```
// Compare the assets present 5 days ago with the assets present in the last day
http://localhost:9090/model/assets/diff?before=1d offset 5d&after=1d
```

**Review by michaelmdresser — COMMENTED** [review]
Nice work! I have a couple of suggestions, and then I'll be ready to approve. Make sure to incorporate any feedback from @jjarrett21, if he has any.

**Comment by Adam-Stack-PM** [comment]
@nickcurie, Just a heads up added the v1.96 to help us keep track of all items worked on during the v1.96 window.

**Comment by nealormsbee** [comment]
@nickcurie hypothetical question for you. Suppose I pass `?aggregate=service&filterServices=Kubernetes`, and my `Kubernetes` service has had individual items added *and* removed over the course of the interval, what do I get back? My guess would be a single `Entity` with either an `Added` or `Removed` kind, depending on the net increased/decreased spend, but seems like something worth asking.

**Comment by nickcurie** [comment]
@nealormsbee The API is comparing 2 asset sets from 2 different windows, so all the added and removed assets will be tracked and displayed using a query with those parameters

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**